### PR TITLE
Add Ground Branch Egg

### PR DESCRIPTION
**Pull request by lilkingjr1**
# Description

- Adds Ground Branch egg and README.
- Updates all directory READMEs to include it (and even fixes some missing entries that surrounded it).

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

## New egg Submissions

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your egg use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [x] Have you added a unique README.md for the egg you are adding according to the alphabetical order?
5. [x] You verify that the start command applied does not use a shell script
    * [ ] If some script is needed then it is part of a current yolk or a PR to add one
6. [x] The egg was exported from the panel

### CODE REVIEW

File: game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            "default_value": "16",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1",
+            "field_type": "text"
+        },
+        {
+            "name": "[Repair] Validate Server Files",
+            "description": "Leave empty (no value) for OFF or type \"true\" or \"1\" for ON. Validates all server files when Automatic Updates is enabled. Note: This will significantly increase server startup times, so it is recommended to only enable this when needed.",
+            "env_variable": "VALIDATE",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json:74]: where is this used/ called?

Comment (lilkingjr1) [reply game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json:74]: https://github.com/parkervcp/yolks/blob/c9dc5173fcab288d40aece96a00e16937288b16d/wine/entrypoint.sh#L29

File: game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
...server banner in-game where possible.\r\n; (recommended resolution: 788 x 386)\r\nServerWebBanner=\"\"\r\n; e.g. ServerWebPage=www.images.com\/myimage.jpeg\r\n\r\n; Password required to join the server.\r\n;ServerPassword=\r\n\r\n; Password required to join the server as a permanent spectator.\r\n;SpectatorOnlyPassword=\r\n\r\n; Maximum number of players allowed on this server.\r\nMaxPlayers=16\r\nMaxSpectators=0\r\n\r\n; Set default match types for game modes and round limits:\r\n; Match types: Continuous, BestOf, FirstTo, Play, PlayUntilWin, TimeLimit.\r\n; Round limits = number of rounds, except Time Limit match type (minutes).\r\n; PVE \/ Co-op:\r\nPVEMatchType=Continuous\r\nPVERoundLimit=2\r\n; PVP:\r\nPVPMatchType=Continuous\r\nPVPRoundLimit=3\r\n; PVP FFA (Free-for-all):\r\nPVPFFAMatchType=Continuous\r\nPVPFFARoundLimit=1\r\nGameRules=((\"AllowCheats\", False),(\"AllowDeadChat\", True),(\"AllowUnrestrictedRadio\", False),(\"AllowUnrestrictedVoice\", False),(\"SpectateEnemies\", True),(\"SpectateForceFirstPerson\", False),(\"SpectateFreeCam\", True),(\"UseTeamRestrictions\", False))\r\nEOF\r\nfi\r\n\r\necho -e \"\\nGround Branch Dedicated Server successfully installed!\\n\"",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json:25]: you you move this config generating to a dwonload of github as we like to keep the install script simple and readable

File: game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        {
+            "name": "[System] WINEPATH",
+            "description": "Used for compatibility. Cannot be changed.",
+            "env_variable": "WINEPATH",
+            "default_value": "\/home\/container",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:\/home\/container",
+            "field_type": "text"
+        }
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/ground_branch/egg-ground-branch.json:130]: does it preform any beter or spit out less errors when the winetrick dotnet46, vcrun2015, 2017, 2019 is installed? ( to stop hiding the errors set WINEDEBUG to `+all` or just empty)
image

**Comment by QuintenQVD0** [comment]
Screenshot_20230608_194227_Discord

**Comment by lilkingjr1** [comment]
Interesting... Good find. I'll work on this when I have some more time.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**